docs/_content/themas/achtergronddocumentatie/versies-en-migraties.md

```diff
@@ -44,25 +48,24 @@ HTTP 200
 }
 ```
 
-1. Huidige situatie: Zaken API v1.0.0
-2. Een Zaak heeft een Status (beide in de Zaken API)
-3. Zaken API v2.0.0 komt uit
+3. De provider heeft er voor gekozen de URI's naar gerelateerde objecten in een koppeltabel in de database vast te leggen.
```

Comment:
Maak je het hiermee niet lastiger juist om te beschrijven wat er gebeurt?

Vind dat "koppeltabel voor gerelateeerde objecten" wel wat uitleg mag hebben.

Reply:
Ik wil voorkomen dat de conclusie wordt getrokken dat wij de voorkeur geven aan een specifieke oplossing. Dat probeer ik te doen door in de verschillende situatieschetsen andere oplossingen als uitgangspunt te nemen.

Ik zal kijken of ik wat extra uitleg op kan nemen m.b.t. de koppeltabel.